lib/config/presets/internal/default.ts

```diff
@@ -456,6 +460,17 @@ export const presets: Record<string, Preset> = {
       'Apply labels <code>{{arg0}}</code> and <code>{{arg1}}</code> to PRs',
     labels: ['{{arg0}}', '{{arg1}}'],
   },
+  labelUpdateTypes: {
+    description:
+      'Apply label per update type, prefixed by <code>{{arg0}}</code> and suffixed by <code>{{arg1}}</code> to PRs',
+    packageRules: options
+      .find((it) => it.name === 'matchUpdateTypes')
+      .allowedValues.map((t) => t as UpdateType)
+      .map((t) => ({
+        matchUpdateTypes: [t],
+        labels: ['{{arg0}}' + t + '{{arg1}}'],
```

Comment:
This will override any previous labels.

Reply:
How is this different from other presets? I'm creating the same objects as its siblings, right?

Reply:
@viceice Shouldn't we fix all current templates from using `labels` to `addLabels`?

Reply:
🤷‍♂️ @rarkins WDYT?